planner/core/stats.go

```diff
@@ -160,7 +160,7 @@ func (ds *DataSource) getColumnNDV(colID int64) (ndv float64) {
 	hist, ok := ds.statisticTable.Columns[colID]
 	if ok && hist.Count > 0 {
 		factor := float64(ds.statisticTable.Count) / float64(hist.Count)
-		ndv = float64(hist.NDV) * factor
+		ndv = float64(hist.Histogram.NDV) * factor
```

Comment:
Could you tell me why add `Histogram`? For me, both of these are fine. I just want to know if there are any more considerations for this change.

Reply:
Because there is an NDV function in FMSketch. This will cause ambiguity.